src/bebone/gfx/window/window.cpp

```diff
@@ -111,4 +113,10 @@ namespace bebone::gfx {
     f32 Window::get_aspect() const {
         return static_cast<f32>(width) / static_cast<f32>(height);
     }
+
+    void Window::end_frame() {

```

Comment:
![image](https://github.com/Maksasj/bebone/assets/36482401/64009ff9-8a51-402e-bd75-2060bed31371)
maybe this code, and these fields to abstract somehow
```
double start_frame_time;
double end_frame_time;
```

I thought about Timer structure or something like that, idk. What do you think ?

Reply:
Ok, I will try to come up with something